### PR TITLE
EKS Container Insights (Cloudwatch Logs)

### PR DESCRIPTION
**Pull request by chrissng**

### CODE REVIEW

File: eks-container-insights/templates/daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: extensions/v1beta1
```
Comment (lawliet89) [inline eks-container-insights/templates/daemonset.yaml]:
```suggestion
apiVersion: apps/v1
```

File: eks-container-insights/templates/daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: {{ include "eks-container-insights.name" . }}
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: {{ include "eks-container-insights.name" . }}
+      annotations:
+        configHash: 8915de4cf9c3551a8dc74c0137a3e83569d28c71044b0359c2578d2e0461825
```
Comment (lawliet89) [inline eks-container-insights/templates/daemonset.yaml]: Shouldn't this be hashed from the contents of the ConfigMap or something?

File: eks-container-insights/templates/rbac.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (lawliet89) [inline eks-container-insights/templates/rbac.yaml]:
```suggestion
apiVersion: rbac.authorization.k8s.io/v1
```

File: eks-container-insights/templates/rbac.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  name: {{ .Values.serviceAccountName }}-role
+rules:
+  - apiGroups: [""]
+    resources:
+      - namespaces
+      - pods
+      - pods/logs
+    verbs: ["get", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (lawliet89) [inline eks-container-insights/templates/rbac.yaml]:
```suggestion
apiVersion: rbac.authorization.k8s.io/v1
```

File: eks-container-insights/templates/daemonset.yaml (hunk 2 above)
Comment (chrissng) [reply eks-container-insights/templates/daemonset.yaml]: I'd remove this. It was copied raw from aws docs

Comment (chrissng) [reply eks-container-insights/templates/daemonset.yaml]: I calculate the hash of rendered configmap

**Pull request merged**